shell/platform/tizen/flutter_tizen_engine.h

```diff
@@ -70,6 +70,8 @@ class FlutterTizenEngine : public TizenRenderer::Delegate {
   bool RunEngine(const FlutterDesktopEngineProperties& engine_properties);
   bool StopEngine();
 
+  void RestartEngine();
```

Comment:
I think RestartEngine is an ambiguous name. First of all, there is no API called start, and This implementation does nothing to the FlutterEngine instance.

Reply:
Hmm..Then.. How about ``reset``?

Reply:
It's reverted